packages/flutter/lib/src/material/text_field.dart

```diff
@@ -191,6 +194,10 @@ class TextField extends StatefulWidget {
 
   /// {@macro flutter.widgets.editableText.maxLines}
   final int maxLines;
+  
+  /// If [maxLength] is set to this number, it disables the showing of the maximum length part of
+  /// the display, yet still gives a character counter.
+  static const int noMaxLength = 9007199254740992; // math.pow(2, 53);
```

Comment:
This doesn't seem like the right approach

Reply:
@jonahwilliams -- It was the approach suggested by @HansMuller in the Issue:

[Issue](https://github.com/flutter/flutter/issues/23791)

Reply:
sounds good then